### PR TITLE
Update Regression Test Suite

### PR DESCRIPTION
**Pull request by ashleywntrbjss**
This PR:

Updates the selenium chromedriver to work with the latest version of chrome

Updates the regression test suite to account for component changes

Updates the page loaded common action to tell you which page it was expecting rather than just ' false '

Splits some tests out into separate suites to allow running in parallel (though there seems to be a login issue - the first test will fail, all others will pass, and the first test will pass on re run)

Updates the test data to ensure that the single solution filter actually returns a single solution

### CODE REVIEW

File: tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Competitions/CompetitionPages.cs [hunk 1]
```diff
@@ -126,6 +126,7 @@ public void StepOnePrepareCompetition(FilterType filterType, string competitionN
                 StartCompetition.CreateCompetition(competitionName);
                 SelectSolutions.AddSolutions(solutions);
                 SolutionShortlisted.SolutionNotIncludedInShortlisting();
```
Comment (dssherlock) [inline tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Competitions/CompetitionPages.cs]: Revert this file

File: tests/NHSD.GPIT.BuyingCatalogue.E2ETests.Framework/Actions/Common/CommonActions.cs [hunk 2]
```diff
@@ -382,13 +384,20 @@ public bool PageLoadedCorrectGetIndex(
             var actionUrl = new Uri("https://www.fake.com/" + absoluteRoute, UriKind.Absolute);
```
Comment (dssherlock) [inline tests/NHSD.GPIT.BuyingCatalogue.E2ETests.Framework/Actions/Common/CommonActions.cs:385]: Isn't this the check to determine if the test fails or passes? Hence it should return false, leading to the test failing gracefully. It should only throw an exception when something unexpected has happened.

File: tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/LocalWebApplicationFactory.cs [hunk 3]
```diff
@@ -33,7 +33,7 @@ public sealed class LocalWebApplicationFactory : WebApplicationFactory<Startup>
 
         // Need to find a better way of doing this
         [SuppressMessage("StyleCop.CSharp.NamingRules", "SA1310:Field names should not contain underscore", Justification = "This name is used by the Webapp, so needs to be kept")]
```
Comment (dssherlock) [inline tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/LocalWebApplicationFactory.cs:36]: This isn't necessary

File: tests/NHSD.GPIT.BuyingCatalogue.E2ETests.Framework/Actions/Common/CommonActions.cs (hunk 2 above)
Comment (ashleywntrbjss) [reply tests/NHSD.GPIT.BuyingCatalogue.E2ETests.Framework/Actions/Common/CommonActions.cs:385]: This is a bit of a makeshift way to get extra information out of this check without having to refactor every instance where its used. At the moment it just returns false and doesn't really tell you what went wrong. With the more detailed output I was able to tell that it was still in the process of logging in when it failed

File: tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/LocalWebApplicationFactory.cs (hunk 3 above)
Comment (ashleywntrbjss) [reply tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/LocalWebApplicationFactory.cs:36]: For me localhost didn't work - I tried to keep it separate and uncommitted but I kept accidentally including it in things so I was hoping I'd be able to make the change here, then as long as it works for everyone else that should be the problem solved

File: tests/NHSD.GPIT.BuyingCatalogue.E2ETests.Framework/Actions/Common/CommonActions.cs (hunk 2 above)
Comment (dssherlock) [reply tests/NHSD.GPIT.BuyingCatalogue.E2ETests.Framework/Actions/Common/CommonActions.cs:385]: Does it not show up in the stack trace?

**Review by dssherlock — DISMISSED** [review]

**Review by bjss-colin — APPROVED** [review]

**Pull request merged**